inference/serial_block_alignment.py

```diff
@@ -18,7 +18,7 @@
 from args import get_argparser, parse_args, get_aligner, get_bbox, get_provenance
 from os.path import join
 from cloudmanager import CloudManager
-
+from time import time
```

Comment:
I have time printouts for each task in the latest `serial_block_alignment`. You can drop all the changes to this file.